app/src/main/java/org/thoughtcrime/securesms/webrtc/CallManager.kt

```diff
@@ -363,7 +371,8 @@ class CallManager(
             val byteArray = ByteArray(buffer.data.remaining()) { buffer.data[it] }
             val json = Json.parseToJsonElement(byteArray.decodeToString()) as JsonObject
             if (json.containsKey("video")) {
-                _remoteVideoEvents.value = VideoEnabled((json["video"] as JsonPrimitive).boolean)
+                _videoState.value = _videoState.value.copy(remoteVideoEnabled = (json["video"] as JsonPrimitive).boolean)
```

Comment:
Do you like this syntax?

```suggestion
                _videoState.update { it.copy(remoteVideoEnabled = (json["video"] as JsonPrimitive).boolean) }
```

Reply:
Yeah that's great